### PR TITLE
chore: made stronger typings, enabled noImplicitAny

### PR DESCRIPTION
**Pull request by DxCx**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -121,27 +119,27 @@ export default class Client {
       this.waitingSubscriptions[subId] = true;
       setTimeout( () => {
         if (this.waitingSubscriptions[subId]){
-          handler(new Error('Subscription timed out - no response from server'));
+          handler([new Error('Subscription timed out - no response from server')]);
```
Comment (DxCx) [inline src/client.ts]: Note that a bug found and fixed in here.
handler used once with Error and then with Error[].
changed to Error[].

**Comment by helfer** [comment]
Thanks @DxCx 😃

**Comment by DxCx** [comment]
Sure @helfer :)
Will be happy to get it published so i can download via npm ;)

**Comment by helfer** [comment]
of course! there's a few more PRs to merge, but you'll have a new version tomorrow.

**Comment by DxCx** [comment]
Thanks! :)

**Pull request merged**